### ISSUE TITLE
BCDT coingecko identifier was changed

### ISSUE THREAD

**Issue by evilny0** [rotki/rotki#2963]
BCdiploma/EvidenZ was rebranded, and the token was renamed. The ticker (BCDT) didn't change, but the name did, and as part of it, the coingecko identifier was changed from "bcdiploma" to "blockchain-certified-data-token".

**Comment by yabirgb** [comment]
@evilny0 Thank you!

**Comment by LefterisJP** [comment]
Why on earth would coingecko change their asset ids is beyond me.

@yabirgb this is not the only one that needs fixing. When I said to take care of the assets I meant to make sure the assets test don't have any warnings. They do and it also includes this asset the user mentions.

```
python pytestgeventwrapper.py -xs  rotkehlchen/tests/unit/test_assets.py::test_coingecko_identifiers_are_reachable
```

has a bunch of warnings. Please address them @yabirgb for v1.17.0

**Comment by LefterisJP** [comment]
This is going to be fixed in the new assets update coming with v1.17.0: https://github.com/rotki/assets/blob/d35b34152da774fed4a404c64b8f7e760b3ccc35/updates/3/updates.sql#L83
